python/servo/build_commands.py

```diff
@@ -618,12 +626,14 @@ def build(self, target=None, release=False, dev=False, jobs=None,
                     return rv
 
             if sys.platform == "win32":
-                servo_exe_dir = os.path.dirname(self.get_binary_path(release, dev, target=target))
+                servo_exe_dir = os.path.dirname(
+                    self.get_binary_path(release, dev, target=target, simpleservo=libsimpleservo)
+                )
                 assert os.path.exists(servo_exe_dir)
 
                 # on msvc builds, use editbin to change the subsystem to windows, but only
                 # on release builds -- on debug builds, it hides log output
-                if not dev:
+                if not dev and libsimpleservo:
```

Comment:
Nit: I find this hard to read because I've not internalized Python's precedence rules.

Reply:
Nit: You found this hard to read because it was entirely incorrect.